CSharp/BatchExplorer/Models/TaskModel.cs

```diff
@@ -51,6 +51,12 @@ public class TaskModel : ModelBase
         public string CommandLine { get { return this.Task.CommandLine; } }
 
         /// <summary>
+        /// The exit code of the task
+        /// </summary>
+        [ChangeTracked(ModelRefreshType.Basic)]
+        public int? ExitCode { get { return this.Task.ExecutionInformation.ExitCode; } }
```

Comment:
It's possible for a task to not have an `ExecutionInformation` property which would trigger a null reference exception here -- can you do this instead?
`this.Task?.ExecutionInformation?.ExitCode`